docs/intro/version_history.rst

```diff
@@ -1,6 +1,74 @@
+  rate variance, and all parameters are now configurable in both runtime and in the JSON
+  configuration.
+* TCP level :ref:`listener <config_listeners_per_connection_buffer_limit_bytes>` and
+  :ref:`cluster <config_cluster_manager_cluster_per_connection_buffer_limit_bytes>` connections now
+  have configurable buffer limits at which point connection level back pressure is applied. Full end
+  to end flow control will be available in a future release.
+* :ref:`Redis health checking <config_cluster_manager_cluster_hc>` has been added as an active
+  health check type. Full Redis support will be documented/supported in 1.4.0.
+* :ref:`TCP health checking <config_cluster_manager_cluster_hc_tcp_health_checking>` now supports a
+  "connect only" mode that only checks if the remote server can be connected to without
+  writing/reading any data.
+* `BoringSSL <https://boringssl.googlesource.com/boringssl>`_ is now the only supported TLS provider.
+  The default cipher suites and ECDH curves have been updated with more modern defaults for both
+  :ref:`listener <config_listener_ssl_context>` and
+  :ref:`cluster <config_cluster_manager_cluster_ssl>` connections.
+* The `header value match` :ref:`rate limit action
+  <config_http_conn_man_route_table_rate_limit_actions>` has been expanded to include an *expect
+  match* parameter.
+* Route level HTTP rate limit configurations now do not inherit the virtual host level
+  configurations by default. The :ref:`include_vh_rate_limits
+  <config_http_conn_man_route_table_route_include_vh>` to inherit the virtual host level options if
+  desired.
+* HTTP routes can now add request headers on a per route and per virtual host basis via the
+  :ref:`request_headers_to_add <config_http_conn_man_route_table_route_add_req_headers>` option.
+* The :ref:`example configurations <install_ref_configs>` have been refreshed to demonstrate the
+  latest features.
+* :ref:`per_try_timeout_ms <config_http_conn_man_route_table_route_retry>` can now be configured in
+  a route's retry policy in addition to via the :ref:`x-envoy-upstream-rq-per-try-timeout-ms
+  <config_http_filters_router_x-envoy-upstream-rq-per-try-timeout-ms>` HTTP header.
+* :ref:`HTTP virtual host matching <config_http_conn_man_route_table_vhost>` now includes support
+  for wildcard domains.
+* The default for tracing random sampling has been changed to 100% and is still configurable in
+  :ref:`runtime <config_http_conn_man_runtime>`.
+* :ref:`HTTP tracing configuration <config_http_conn_man_tracing>` has been extended to allow tags
+  to be populated from arbitrary HTTP headers.
+* The :ref:`HTTP rate limit filter <config_http_filters_rate_limit>` can now be applied to internal,
+  external, or all requests via the `request_type` option.
+* :ref:`Listener binding <config_listeners>` now requires specifying an `address` field. This can be
+  used to bind a listener to both a specific address as well as a port.
```

Comment:
The breaking change policy starts at 1.3.0, so I'm not going to list out every breaking change. I will clarify the text a bit to make this clear.